mora/auth/auth.py

```diff
@@ -56,6 +57,20 @@ def __call__(self, r):
         return r
 
 
+@blueprint.route('/user', methods=['GET'])
+def get_user():
+    '''Get the currently logged in user
+
+    .. :quickref: Authentication; Get user
+
+    :return: The username of the user who is currently logged in.
+    '''
+
+    username = flask.session.get('username') or 'N/A'
```

Comment:
Wouldn't it be better to return `None` or `null` when not logged in, and then have the frontend handle it?